gate-core/gate-core.gradle

```diff
@@ -15,6 +15,7 @@
  */
 
 dependencies {
+  api "org.springframework.boot:spring-boot-starter-thymeleaf"
```

Comment:
Why is this dependency being introduced?

Reply:
Removed this dependency.